lib/spectator_sport/engine.rb

```diff
@@ -13,6 +13,7 @@ class Engine < ::Rails::Engine
   module Dashboard
     class Engine < ::Rails::Engine
       isolate_namespace SpectatorSport::Dashboard
+      paths.add "config/routes.rb", with: "config/dashboard_routes.rb"
```

Comment:
A little strange, but it checks out.

In the docs: https://github.com/bensheldon/rails/blob/0d30e84878223df68efda3b0e1741611d9682a45/railties/lib/rails/paths.rb#L35-L35

Where the config is used: https://github.com/bensheldon/rails/blob/0d30e84878223df68efda3b0e1741611d9682a45/railties/lib/rails/engine.rb#L592-L592